growattServer/__init__.py

```diff
@@ -59,9 +63,9 @@ def __get_date_string(self, timespan=None, date=None):
 
         return date_str
 
-    def get_url(self, page):
+    def __get_url(self, page):
```

Comment:
For backward compatibility i prefer to keep it as
```suggestion
    def get_url(self, page):
```

I'm not sure if/how other people use it, so that helps break as little things as possible 🙂